libs/ktem/ktem/db/base_models.py

```diff
@@ -95,3 +96,78 @@ class BaseIssueReport(SQLModel):
     chat: Optional[dict] = Field(default=None, sa_column=Column(JSON))
     settings: Optional[dict] = Field(default=None, sa_column=Column(JSON))
     user: Optional[int] = Field(default=None)
+
+
+class TagType(str, Enum):
+    text = "Text"
+    classification = "Classification"
+    boolean = "True/False"
+
+    @staticmethod
+    def get_types() -> list[str]:
+        return [
+            TagType.text.value,
+            TagType.classification.value,
+            TagType.boolean.value,
+        ]
```

Comment:
```suggestion
    @classmethod
    def values(cls) -> Sequence[str]:
        return [en.value for en in cls]
```
actually this values could be a class property, however python does not provide an official support for this one, could be a lil bit tricky